lib/matplotlib/markers.py

```diff
@@ -25,12 +25,14 @@
 `"*"`                          star
 `"h"`                          hexagon1
 `"H"`                          hexagon2
-`"+"`                          plus (filled)
-`"x"`                          x (filled)
+`"+"`                          plus
+`"x"`                          x
 `"D"`                          diamond
 `"d"`                          thin_diamond
 `"|"`                          vline
 `"_"`                          hline
+`"P"`                          plus (filled)
```

Comment:
Keep them together with the non-filled versions?

It may also make sense to reorder the entries a bit (say square / pentagon / hexagon1/2 / octogon) for example.  Also, what is the difference between the two hexagons?  Orientation I guess?